src/components/equations/ItemEffects.ts

```diff
@@ -18,4 +18,41 @@ export class ItemEffects<TGameStartr extends FullScreenPokemon> extends GeneralC
 
         move.remaining = Math.min(move.remaining + amount, move.uses);
     }
+
+    /**
+     * Increases a stat by 1.
+     *
+     * @param statistic   The statistic that is being modified.
+     */
+    private addBattleStat(statistic: IStatistic) {
```

Comment:
General rule of thumb: doc comments should generally mean about the exact same as the method. `addBattleStat` implies we're adding a new battle stat. It should probably be `increaseBattleStat`.